.pre-commit-config.yaml

```diff
@@ -96,7 +96,7 @@ repos:
           - post-rewrite
           - push
   - repo: https://github.com/jumanjihouse/pre-commit-hooks
-    rev: 2.1.6
+    rev: 3.0.0
```

Comment:
This is a good change, but let's save this for a future pull request since it's unrelated to the MegaLinter upgrade.